e2e/resources/workbench-config.ts

```diff
@@ -57,7 +57,7 @@ const test: IEnvConfig = {
 
 // workbench staging environment
 const staging: IEnvConfig = {
-  LOGIN_URL_DOMAIN_NAME: process.env.STAGING_LOGIN_URL || 'https://all-of-us-rw-staging.appspot.com',
+  LOGIN_URL_DOMAIN_NAME: process.env.STAGING_LOGIN_URL || 'https://staging.fake-research-aou.org',
   API_HOSTNAME: process.env.API_HOSTNAME || 'api-dot-all-of-us-rw-staging.appspot.com',
```

Comment:
Does the API_HOSTNAME need to change as well?

Reply:
ahh good point! I see we did change api server address